cloud/amazon/iam_cert.py

```diff
@@ -273,10 +273,15 @@ def main():
                                                     iam.get_all_server_certs().\
                                                     list_server_certificates_result.\
                                                     server_certificate_metadata_list]
-    orig_bodies = [iam.get_server_certificate(thing).\
-                  get_server_certificate_result.\
-                  certificate_body \
-                  for thing in orig_certs]
+    orig_bodies = []
+    for thing in orig_certs:
+        try:
+            orig_bodies.append(iam.get_server_certificate(thing).\
+                               get_server_certificate_result.\
+                               certificate_body)
+        except boto.exception.BotoServerError:
+            continue
```

Comment:
this ignores all errors, not a good way to 'handle' this

a more sensible one would be to use module.fail_json to send a reasonable error to the user or at least capture it and send it as part of warnings